features/step_definitions/debugging_steps.rb

```diff
@@ -10,3 +10,7 @@
 When /^I write the page body to (.*)$/ do |path|
   File.open(path, 'w') { |f| f.puts(page.body) }
 end
+
+And /^(?:|I )confirm alert message is present "([^"]*)"$/ do |msg|
```

Comment:
I don't think we need this extra debugging step. The following step definitions should accomplish the same thing:

```ruby
features/step_definitions/web_steps.rb

When /^(?:|I )press "([^"]*)" and confirm "([^"]*)"$/ do |button, msg|
  accept_confirm msg do
    click_button(button)
  end
end

When /^(?:|I )follow "([^"]*)" and confirm "([^"]*)"$/ do |link, msg|
  accept_confirm msg do
    click_link(link)
  end
end
```